### ISSUE TITLE
Dataset  problem

### ISSUE THREAD

**Issue by 554773162** [ybgdgh/Co-NavGPT#7]
After run code: python -m habitat_sim.utils.datasets_download --username xxx --password xxx --uids hm3d_minival_v0.2
I got: Data download failed, no datasource named hm3d_minival_v0.2

After removing "_v0.2", it works. But the dataset structure is totally different from yours
data/scene_datasets
data/versioned_data

Therefore, i dont know what to do next. Could you give me some good advice, please?

**Comment by ybgdgh** [comment]
Yes it doesn't support downloading v0.2 datatset now. But you can find the download file in their latest version repo, such as [this one](https://github.com/facebookresearch/habitat-sim/blob/v0.2.3/src_python/habitat_sim/utils/datasets_download.py).

**Comment by 554773162** [comment]
> Yes it doesn't support downloading v0.2 datatset now. But you can find the download file in their latest version repo, such as [this one](https://github.com/facebookresearch/habitat-sim/blob/v0.2.3/src_python/habitat_sim/utils/datasets_download.py).

Thank you for your help. Data issue has been closed. But I meet another issue: 
/habitat-sim/habitat-lab/habitat/core/embodied_task.py", line 268, in _init_entities
    assert (
AssertionError: invalid TURN_LEFT_S type TurnLeftAction_S
 
is it beacuse I installed wrong habitat version?

**Comment by 554773162** [comment]
> > Yes it doesn't support downloading v0.2 datatset now. But you can find the download file in their latest version repo, such as [this one](https://github.com/facebookresearch/habitat-sim/blob/v0.2.3/src_python/habitat_sim/utils/datasets_download.py).
> 
> Thank you for your help. Data issue has been closed. But I meet another issue: /habitat-sim/habitat-lab/habitat/core/embodied_task.py", line 268, in _init_entities assert ( AssertionError: invalid TURN_LEFT_S type TurnLeftAction_S
> 
> is it beacuse I installed wrong habitat version?

I've identified the issue: I forgot to replace the habitat folder. However, I'm still concerned about two points:
1. In /Co-NavGPT/exp_main_gpt.py at line 395, within the parse_answer function, there's an error: ValueError: not enough values to unpack (expected 2, got 1). This occurs when trying to split the line with key, value = line.split(': '). I resolved this using a "try...except..." block, but I'm concerned about whether this might impact the results.

2. How to show a graphical user interface just like your demo?

**Comment by ybgdgh** [comment]
1. I think it's because the response of the LLM doesn't match the output format, you can try to request again to get the answer.
2. you can just add a parameter  '-v 1 '  in your command.

**Comment by yczhao10** [comment]
> After run code: python -m habitat_sim.utils.datasets_download --username xxx --password xxx --uids hm3d_minival_v0.2 I got: Data download failed, no datasource named hm3d_minival_v0.2
> 
> After removing "_v0.2", it works. But the dataset structure is totally different from yours data/scene_datasets data/versioned_data
> 
> Therefore, i dont know what to do next. Could you give me some good advice, please?


I encountered the same problem. The dataset structure I obtained after removing "_v0.2" is completely different from yours:

data/scene_datasets
data/versioned_data

 Can this [download file](https://github.com/facebookresearch/habitat-sim/blob/v0.2.3/src_python/habitat_sim/utils/datasets_download.py) from the latest version repository in your reply download the dataset structure that is exactly the same as yours? I don't know how to use this download file.

 I want to obtain the dataset structure that is exactly the same as in Co-NavGPT. Could you please give me some suggestions? Thank you very much! @ybgdgh @554773162
